torch/_functorch/_aot_autograd/utils.py

```diff
@@ -525,24 +525,52 @@ def saved_tensors_hooks_are_inlineable(hooks) -> bool:
 
 def without_output_descs(f: Callable[_P, tuple[_T, _S]]) -> Callable[_P, _T]:
     @wraps(f)
+    @simple_wraps(f)
     def inner(*args, **kwargs):
         return f(*args, **kwargs)[0]
 
     return inner
 
 
+_P = ParamSpec("_P")
+_P2 = ParamSpec("_P2")
+_R = TypeVar("_R")
+_R2 = TypeVar("_R2")
+
+
+def simple_wraps(
+    f: Callable[_P, _R],
+) -> Callable[[Callable[_P2, _R2]], Callable[_P2, _R2]]:
```

Comment:
Why doesn't P = P2 here and R=R2? It's just wraps and doesn't change the param type or the return type right??

Reply:
As it stands, this will erase type info because there is nothing to resolve P2 based on the input or value of P1...